### PR TITLE
10 implement database in code

### LINKED ISSUE #10: Implement database in code
Developer must implement Db Diagram in code.

### PR DESCRIPTION
**Pull request by maryanchuk04**
Realize multilayer architecture.

Create database in server side.

Setup base entities

Add swagger configuration

### CODE REVIEW

**Review by Vova-Romaniuk — COMMENTED** [review]
Cool

File: StudCity.Backend/StudCity.API/Controllers/AuthenticateController.cs [hunk 1]
```diff
@@ -0,0 +1,10 @@
+using Microsoft.AspNetCore.Mvc;
+
+namespace StudCity.API.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class AuthenticateController
+{
+
```
Comment (Serhiy-dotcom) [inline StudCity.Backend/StudCity.API/Controllers/AuthenticateController.cs:9]: Why empty paragraph, and not just simple '{}'

**Review by Serhiy-dotcom — APPROVED** [review]
Maks please fix the issue in file: StudCity.Backend/StudCity.API/Controllers/AuthenticateController.cs
and you can merge then:)

**Review by Vova-Romaniuk — APPROVED** [review]
Cool 😎😎😎

**Pull request merged**